test/e2e/e2ethanos/services.go

```diff
@@ -372,6 +374,11 @@ func (q *QuerierBuilder) WithQueryMode(mode string) *QuerierBuilder {
 	return q
 }
 
+func (q *QuerierBuilder) WithEnableXFunctions(enable bool) *QuerierBuilder {
+	q.enableXFunctions = enable
+	return q
+}
```

Comment:
This may not need an arg, can just set to true,
```suggestion
func (q *QuerierBuilder) WithEnableXFunctions() *QuerierBuilder {
	q.enableXFunctions = true
	return q
}
```